src/cursor/cur_file.c

```diff
@@ -691,6 +691,8 @@ __curfile_cache(WT_CURSOR *cursor)
 
     WT_TRET(__wt_cursor_cache(cursor, cbt->dhandle));
     WT_TRET(__wt_session_release_dhandle(session));
+
+    API_END_STAT(session, ret, cursor_cache);
```

Comment:
Can you please change this one to `API_END_RET_STAT` instead of `API_END_STAT` and remove the follow-on `return` statement?

Reply:
done, thanks.
@kommiharibabu